fotoapparat/src/main/java/io/fotoapparat/hardware/v1/capabilities/CapabilitiesFactory.java

```diff
@@ -71,7 +71,7 @@ public Capabilities fromParameters(Camera.Parameters parameters) {
 		List<String> supportedFlashModes = parameters.getSupportedFlashModes();
 		return supportedFlashModes != null
 				? supportedFlashModes
-				: Collections.<String>emptyList();
+				: Collections.singletonList(Camera.Parameters.FLASH_MODE_OFF);
```

Comment:
I believe that this is the only change which is needed to fix the issue. I suggest we revert everything else.